packages/ses/src/commons.js

```diff
@@ -19,6 +19,7 @@ export { universalThis as globalThis };
 
 export const {
   Array,
```

Comment:
```suggestion
  structuredClone,
  Array,
```
(to avoid being affecting by post-hoc manipulation)

Reply:
To solve the stated problem, line 181 below already says

```js
const { structuredClone } = globalThis;
```

I do it in the ArrayBuffer section as a non-export because `structuredClone` is not part of JS, so I would rather not export it from commons.js

Resolving. But unresolve if you disagree.